### DISCUSSION TITLE
三角督變四腳督

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1648] [General]
台灣的總統璇舉即將在明年舉行，這次有民進黨的類清德，國民黨的侯友宜，民眾黨的柯文哲，即今期宣布參選的郭台銘。
我認為郭台銘在2024年時曾參加國民黨的初選，所以他若出來選勢必會影響侯友宜的選情，而他又有不輸柯文哲的名間號召力，所以勢必也會影響他的選情。
我認為郭台銘，柯文哲跟侯友宜如果想為台灣做事，也想下架民進黨，那他們應該選擇合作，至少我認為郭台銘跟柯文哲可以合作共贏，才能有機會打敗另外兩人。
參考資料:
https://www.hk01.com/%E5%8F%B0%E7%81%A3%E6%96%B0%E8%81%9E/952221/2024%E5%8F%B0%E7%81%A3%E5%A4%A7%E9%81%B8-%E7%A8%B1-%E9%83%AD%E6%9F%AF%E5%90%88-%E9%BB%98%E5%A5%91%E8%BC%83%E5%A5%BD-%E9%83%AD%E5%8F%B0%E9%8A%98%E8%BE%A6%E5%85%AC%E5%AE%A4-%E5%90%88%E4%BD%9C%E7%A9%BA%E9%96%93%E5%A4%A7

**Comment by davidteng00000** [comment]
台灣總統選舉涉及多個候選人，合作可能提高勝算，但涉及政策、策略和政黨支持。但是可能沒有這麼簡單，畢竟台灣人容易受不良媒體哄騙且缺乏媒體識讀素養，可能會因為媒體操作而影響大選結果

**Reply by uwje** [reply]
的確，但若每個候選人的理念近似，且目前狀況很明顯是若繼續分裂，則他們的目標(下架民進黨)根本不可能達成。
希望他們可以基於相同理念而合作，雖然一定會有損失，但總比護送共同討厭的人選上總統來的好
